Initial portfolio setup

Danie12345 opened the pull request:
The first major advance in the portfolio.
Includes:

Header, Headline and Background for mobile.

Danie12345 committed:
Set up repository.

Set up README.md, linters (Lighthouse, Webhint and Stylelint), index.html and index.css, .gitignore and package.json.

Danie12345 committed:
First major advance in portfolio.

index.html:
- Added proper fonts, added text and menu icon.
- Added classes for different styling.
- Added img tags with icons.

index.css:
- Added styles for header, and headline.
- Added background colors and image.
- Stylized different fonts and elements.

Added Icons folder with all icons and images.

Danie12345 committed:
Fixed linter errors

Multiple linter errors fixed in index.css, two errors in index.html.

smunozmo commented on index.html:
```diff
@@ -0,0 +1,44 @@
+</head>
+<body class="body-container">
+    <header class="header">
+        <span class="navitem name">Daniel Malo</span>
```
- [x] Please use an `a` tag for your logo, it should be clickable.

Danie12345 replied:
@smunozmo Sorry, I do not understand. My name is supposed to also be my logo?

Danie12345 replied:
I have considered it so, and changed it

smunozmo commented on index.html:
```diff
@@ -0,0 +1,44 @@
+            </svg>
+        </a>
+    </header>
+    <div class="content-container content decorative-background">
+        <h1>
+            <span>Hey There.</span>
+            <span>I'm Daniel</span>
+        </h1>
+        <section>
+            <h3>I'm a Software Developer</h3>
+            <div class="self-intro">
+                I can help you build a product, feature or website. Look through some of 
+                my work and experience it! If you like what you see and have a project 
+                you need coded, please don't hesitate to contact me.
+            </div>
+        </section>
+        <div class="media-icons">
+            <img class="media-icon" src="Icons/Vector.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-1.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-2.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-3.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-4.png" alt="no image available">
+        </div>
+    </div>
```
- [x]  Nice work here, it would be better if you used a `section` tag instead of a `div` for this section.

smunozmo commented on index.html:
```diff
@@ -0,0 +1,44 @@
+        </h1>
+        <section>
+            <h3>I'm a Software Developer</h3>
+            <div class="self-intro">
+                I can help you build a product, feature or website. Look through some of 
+                my work and experience it! If you like what you see and have a project 
+                you need coded, please don't hesitate to contact me.
+            </div>
```
- [x] This is looking good, please use a `p` tag for this text.

smunozmo commented on index.html:
```diff
@@ -0,0 +1,44 @@
+                you need coded, please don't hesitate to contact me.
+            </div>
+        </section>
+        <div class="media-icons">
+            <img class="media-icon" src="Icons/Vector.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-1.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-2.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-3.png" alt="no image available">
+            <img class="media-icon" src="Icons/Vector-4.png" alt="no image available">
+        </div>
```
- [x] You are almost there, however, it's important to use an `ul` tag here and every icon should be listed with `li` tags and `a` tags to make them clickable, like this:

```
<ul>
    <li><a>content</a></li>
    <li><a>content</a></li>
    <li><a>content</a></li>
</ul>
````

smunozmo reviewed (changes requested):
Hi @Danie12345, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Fixing your linters problem, try to set it up again to avoid a project invalidation in the following reviews. 

Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Danie12345 committed:
Made logo and hamburger menu button clickable

Changed span tag for a tag to make logo clickable in index.html, then reorganized the header elements.

Defined the a tag in index.css.

Danie12345 committed:
Changed headline tag from div to section and indentation fix

Changed the headline tag from using div to section and also indented the contents inside the html tag in index.html

Danie12345 committed:
Changed a div tag for a p tag

Changed the headline main text's tag from div to p, to denote a paragraph in index.html.

Danie12345 committed:
Changes to icons, now clickable

Changed the socials icons for clickable links and changed the tags to be an unordered list with its respective list items in index.html.

Deleted the styling provided by ul and li tags, and fixed the media-icons class to fit the different elements and keep style in index.css.

Danie12345 committed:
Update index.css

Fixed linter error, length-zero-no-unit, replaced vh with percentage.

CollinsTatang reviewed (approved):
Hi @Danie12345, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![image](https://user-images.githubusercontent.com/55596970/158098566-50966056-ce3d-4c4b-aac3-cce7a7d107fb.png)

### **OPTIONAL CHANGES**

- [ ] Please do corrections on your design the background image is overlapping the text and the header cover the heading text.
**Take a look**
![overlap](https://user-images.githubusercontent.com/55596970/158942011-b2c3cbb6-6a3d-4efb-ad45-b789836c917c.png)
**Figmal Design**
![figma](https://user-images.githubusercontent.com/55596970/158942091-438fd7f4-e826-4021-9c24-fbaf5f031259.png)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
